src/main/java/com/adyen/model/checkout/CreatePaymentLinkRequest.java

```diff
@@ -59,24 +79,99 @@
 
+    @SerializedName("lineItems")
+    private List<LineItem> lineItems = null;
+
     @SerializedName("merchantAccount")
     private String merchantAccount = null;
 
+    @SerializedName("merchantOrderReference")
+    private String merchantOrderReference = null;
+
+    @SerializedName("metadata")
+    private Map<String, String> metadata = null;
+
+    /**
+     * Defines a recurring payment type. Allowed values: * &#x60;Subscription&#x60; – A transaction for a fixed or variable amount, which follows a fixed schedule. * &#x60;CardOnFile&#x60; – With a card-on-file (CoF) transaction, card details are stored to enable one-click or omnichannel journeys, or simply to streamline the checkout process. Any subscription not following a fixed schedule is also considered a card-on-file transaction. * &#x60;UnscheduledCardOnFile&#x60; – An unscheduled card-on-file (UCoF) transaction is a transaction that occurs on a non-fixed schedule and/or has variable amounts. For example, automatic top-ups when a cardholder&#x27;s balance drops below a certain amount.
+     */
+    @JsonAdapter(RecurringProcessingModelEnum.Adapter.class)
+    public enum RecurringProcessingModelEnum {
+        CARDONFILE("CardOnFile"),
+        SUBSCRIPTION("Subscription"),
+        UNSCHEDULEDCARDONFILE("UnscheduledCardOnFile");
+
+        @JsonValue
+        private String value;
+
+        RecurringProcessingModelEnum(String value) {
+            this.value = value;
+        }
+
+        public String getValue() {
+            return value;
+        }
+
+        @Override
+        public String toString() {
+            return String.valueOf(value);
+        }
+
+        public static RecurringProcessingModelEnum fromValue(String text) {
+            for (RecurringProcessingModelEnum b : RecurringProcessingModelEnum.values()) {
+                if (String.valueOf(b.value).equals(text)) {
+                    return b;
+                }
+            }
+            return null;
+        }
+
+        public static class Adapter extends TypeAdapter<RecurringProcessingModelEnum> {
+            @Override
+            public void write(final JsonWriter jsonWriter, final RecurringProcessingModelEnum enumeration) throws IOException {
+                jsonWriter.value(enumeration.getValue());
+            }
+
+            @Override
+            public RecurringProcessingModelEnum read(final JsonReader jsonReader) throws IOException {
+                String value = jsonReader.nextString();
+                return RecurringProcessingModelEnum.fromValue(String.valueOf(value));
+            }
+        }
+    }
+
+    @SerializedName("recurringProcessingModel")
+    private RecurringProcessingModelEnum recurringProcessingModel = null;
+
     @SerializedName("reference")
     private String reference = null;
 
     @SerializedName("returnUrl")
     private String returnUrl = null;
 
+    @SerializedName("reusable")
+    private Boolean reusable = null;
+
+    @SerializedName("riskData")
```

Comment:
the `riskData` field seems to appear only from v65 on the specs

Reply:
sorry wrong heading, the models are updated from v66

Reply:
Client.java points to v64, https://github.com/Adyen/adyen-java-api-library/blob/5e2ac0a0ae3fe86077785711e6f9721b660e5404/src/main/java/com/adyen/Client.java#L51
so I am not sure if this field will be considered, can you create a ticket to update checkout to v65? (v66 doesn't appear on our public API explorer yet)

Reply:
Yes, they are new fields so will be skipped in current version. I have created ticket to update checkout to v65.